### PR TITLE
Issue 3 usdnb to nsbt migration 0to6 dec

### PR DESCRIPTION
**Pull request by DeFi-Evangelist**

### CODE REVIEW

File: script/liquidation.ride [hunk 1]
```diff
@@ -71,16 +71,14 @@ func getLastOrderOwnerKey(owner: String) = "last_order_owner_" + owner
...
-func convertWavesToBond(amount: Int, price: Int) = convertNeutrinoToBond(convertWavesToNeutrino(amount, price))
+func convertWavesToBond(amount: Int, price: Int) = convertWavesToNeutrino(amount, price)
 
 #-------------------Global vars -------------------------
 let neutrinoContract = addressFromStringValue(getStringByKey(NeutrinoContractKey))
 let controlContract = addressFromStringValue(getStringByAddressAndKey(neutrinoContract, ControlContractKey)) ## The value from neutrino.ride
 let liquidationContract = this
 let neutrinoAssetId = fromBase58String(getStringByAddressAndKey(neutrinoContract, NeutrinoAssetIdKey)) # Assets ids. The value from neutrino.ride
-let bondAssetId = fromBase58String(getStringByAddressAndKey(neutrinoContract, BondAssetIdKey))
+let bondAssetId = fromBase58String("6nSpVyNH7yM69eg446wrQR94ipbbcmZMU1ENPwanC97g") # NSBT with 6 decimals as USDN does
```
Comment (CryptoBadBoy) [inline script/liquidation.ride]: Add validate min amount in addLiquidationOrder

File: script/neutrino.ride [hunk 2]
```diff
@@ -268,6 +289,48 @@ func transfer(account: String) = {
...
+    else if (migrationStatus == finalStatus)
+        then throw("error: order has already migrated")
+    else if (lastOrder == lastMigratedOrderId)
+        then throw("status ok: all order has already migrated or nothing to migrate")
+    else ScriptResult(
+        WriteSet([
+            DataEntry(keyStatusLiquidationMigration, finalStatus),
+            DataEntry(keyLastMigratedOrder, currentOrderId)
+        ]),
+        TransferSet([ScriptTransfer(orderOwner, orderAmount*PAULI, bondAssetId)])
```
Comment (amurdev) [inline script/neutrino.ride]: Need to transfer new NSBT to liquidation account instead of order owner

File: script/neutrino.ride [hunk 3]
```diff
@@ -246,16 +267,16 @@ func withdraw(account : String, index: Int) = {
 # [called by pacemaker oracles]
 @Callable(i)
 func transferToAuction() = {
-    let deficitBondAmount = convertNeutrinoToBond(deficit) - assetBalance(addressFromStringValue(auctionContract), bondAssetId)
-    let minDeficit = convertNeutrinoToBond(neutrinoSupply)*DEFICITOFFSET/100
+    let deficitBondAmount = deficit - assetBalance(addressFromStringValue(auctionContract), bondAssetId)
```
Comment (amurdev) [inline script/neutrino.ride]: Minor difference between old and new implementations, see [comment](https://gitlab.com/neutrino_protocol/neutrino-contract/-/issues/3#note_302487022 )

File: script/neutrino.ride [hunk 4]
```diff
@@ -144,6 +143,28 @@ func getUnlockBalanceBlock(owner : String) = getNumberByKey(getBalanceUnlockBloc
...
+func getOrderStatusKey(orderId: String) = "order_status_" + orderId
+func getOrderFilledTotalKey(orderId: String) = "order_filled_total_" + orderId
+
+func getOrderTotal(id : String) = getNumberByAddressAndKey(liquidationContract, getOrderTotalKey(id))
+func getOrderOwner(id : String) = getStringByAddressAndKey(liquidationContract, getOrderOwnerKey(id))
+func getOrderStatus(id : String) = getStringByAddressAndKey(liquidationContract, getOrderStatusKey(id))
+func getOrderFilledTotal(id : String) = getNumberByAddressAndKey(liquidationContract, getOrderFilledTotalKey(id))
+func getOrderPrev(id: String) = getStringByAddressAndKey(liquidationContract, getOrderPrevKey(id))
+func getOrderNext(id: String) = getStringByAddressAndKey(liquidationContract, getOrderNextKey(id))
+
```
Comment (amurdev) [inline script/neutrino.ride]: remove unused funcs and vars

**Pull request merged**